### ISSUE TITLE
Latest version of asynq 0.24.1 breaks asynqmon

### ISSUE THREAD

**Issue by abezzub** [hibiken/asynqmon#302]
I am getting following error:
```
panic: asnyqmon.New: unsupported RedisConnOpt type asynq.RedisClientOpt

goroutine 1 [running]:
github.com/hibiken/asynqmon.New({{0x10580ef4e, 0xd}, {0x1065729c0, 0x140001d9180}, {0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, ...})
```
Error originates from here:
```
rc, ok := opts.RedisConnOpt.MakeRedisClient().(redis.UniversalClient)
if !ok {
	panic(fmt.Sprintf("asnyqmon.New: unsupported RedisConnOpt type %T", opts.RedisConnOpt))
}
```
https://github.com/hibiken/asynqmon/blob/master/handler.go#L66

When I revert my project to use Asynq 0.24.0 the error goes away. I am using v9 Redis client.

**Comment by missish** [comment]
func Monitor() {
	h := asynqmon.New(asynqmon.Options{
		RootPath:     "/", // RootPath specifies the root for asynqmon app
		RedisConnOpt: asynq.RedisClientOpt{Addr: ":6379", Password: "", DB: 0},
	})

	e := echo.New()
	e.Any("/*", echo.WrapHandler(h))
	e.Start(":8080")
}

Visit http://localhost:8080/ why this error occurs
**template: pattern did not match any files: `ui\build\index.html`**
How can I solve it?
@anupriya17

**Comment by Joker666** [comment]
+1

**Comment by trungdlp-wolffun** [comment]
+1

**Comment by hibiken** [comment]
Should be fixed via #303

**Comment by windowshopr** [comment]
+1 template: pattern matches no files: `ui\build\index.html`
